<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplinePropertyWidget</class>
 <widget class="QWidget" name="SplinePropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="show3DWidget">
     <property name="text">
      <string>Show Widget</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="pqTreeWidget" name="handlePositions">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Z</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="vboxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addPoint">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
         <normaloff>:/QtWidgets/Icons/pqPlus16.png</normaloff>:/QtWidgets/Icons/pqPlus16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removePoints">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
         <normaloff>:/QtWidgets/Icons/pqDelete16.png</normaloff>:/QtWidgets/Icons/pqDelete16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="closed">
     <property name="text">
      <string>Closed Spline</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drag control points to change point locations.&lt;br/&gt;Use Ctrl + Left Click to insert a new point on the line at mouse position.&lt;br/&gt;Use Shift + Left Click to remove the point at mouse position.&lt;br/&gt;Use Alt + Left Click to add a new point anywhere at mouse position&lt;br/&gt;at the end of the polyline or at the beginning if the first point is selected.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
